google-cloud-contrib/google-cloud-nio/src/test/java/com/google/cloud/storage/contrib/nio/CloudStorageFileSystemTest.java

```diff
@@ -159,4 +161,27 @@ public void testListFiles() throws IOException {
       assertThat(got).containsExactlyElementsIn(goodPaths);
     }
   }
+
+  @Test
+  public void testMatcher() throws IOException {
+    try (FileSystem fs = CloudStorageFileSystem.forBucket("bucket")) {
+      String pattern1 = "glob:*.java";
+      PathMatcher matcher1 = fs.getPathMatcher(pattern1);
+      String pattern2 = "glob:*.{java,text}";
+      PathMatcher matcher2 = fs.getPathMatcher(pattern2);
```

Comment:
for clarity: call `javaAndTextFileMatcher` and move down after the asserts for the first matcher so that it is closer to its first use.